packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts

```diff
@@ -241,6 +241,9 @@ export default class NotificationServicesController extends BaseController<
   NotificationServicesControllerState,
   NotificationServicesControllerMessenger
 > {
+  // Temporary boolean as push notifications are not yet enabled on mobile
+  #isPushIntegrated = true;
```

Comment:
Temp for mobile. We will remove this in a later release

Reply:
Push Notification controllers isn't integrated and won't be on Mobile since MM app relies on Firebase SDK directly. Maybe we should remove the comment and use this flag as a feature that it is.

Reply:
OK fair, as this is 0.x.x packages, there most likely will be more changes coming down the pipeline. I shall update any documentation and comments in a future version.